### PR TITLE
VIM-4824: Three sixty model

### PR DESCRIPTION
**Pull request by jenniferlim**
## Ticket

https://vimean.atlassian.net/browse/VIM-4824

## Ticket Summary

Retrieve Spatial object for 360

## Implementation Summary

Added a new object in VimeoNetworking to retrieve the Spatial datas.
Updated the filter result in VideoStream to accept to retrieve the Spatial datas.

## How to Test
TBD

### CODE REVIEW

File: VimeoNetworking/Sources/Models/Spatial.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//
+//
+//
+
+/// Spatial stores all information related to threesixty video
+public class Spatial: VIMModelObject
+{
+    
+    /// Represents the projection. Value returned by the server can be: "equirectangular", "cylindrical", "cubical", "pyramid", "dome".
+    dynamic var projection: String?
```
Comment (jasonhawkins) [inline VimeoNetworking/Sources/Models/Spatial.swift]: `dynamic` so we can KVO these properties from Objective-C?

File: VimeoNetworking/Sources/Models/Spatial.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    
+    /// Represents the format. Value returned by the server can be: "mono", left-right", "top-bottom"
```
Comment (jasonhawkins) [inline VimeoNetworking/Sources/Models/Spatial.swift]: Awesome, hooray for documentaaaationnnnn

File: VimeoNetworking/Sources/Models/Spatial.swift (hunk 1 above)
Comment (jenniferlim) [inline VimeoNetworking/Sources/Models/Spatial.swift]: that was the idea, but finally i don't think we need to kvo this. I'll update the PR.

Comment (jasonhawkins) [inline VimeoNetworking/Sources/Models/Spatial.swift]: I guess if we ever need to take advantage of KVO for these then we can change it back. In the meantime we'll enjoy that sweet, sweet minuscule and imperceptible performance improvement by not making it dynamic! Woooo 🏃🏻‍♀️💨

**Comment by jenniferlim** [comment]
@jasonhawkins - Thanks for your feedback, I updated my PR. Let me know if we are good to 📫 ?

**Comment by jasonhawkins** [comment]
I think you're good to go here, thanks Jennifer 👍🏼

**Comment by jenniferlim** [comment]
Awesome ⭐️

**Pull request merged**